test/src/post-make.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+function main() {
```

Comment:
This is an interesting style for shell functions I haven't seen anywhere else. The two styles I've seen are:

- `function main {` (note the lack of parentheses)
- `main() {`

Among these, I heavily prefer the latter. It seems to be the most widespread one in my experience and it is consistent with what we do in Homebrew code. What do you think?

On a side note: Would you mind converting this to a two-space indent with spaces, like we tend to favor in all other Homebrew code I'm aware of (with `Makefile`s being an obvious exception)? You might want to tweak the `.editorconfig` for that.